packages/generator/src/sdk-metadata/code-samples/generic-get-all-code-sample.ts

```diff
@@ -0,0 +1,3 @@
+import { TestEntity } from '@sap-cloud-sdk/core/test/test-util/test-services/v2/test-service';
```

Comment:
The code example should only import from root index
```suggestion
import { TestEntity } from '@sap-cloud-sdk/core'
```